package/firmware/renesas-usb-firmware/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=upd72020x-fw
+PKG_SOURCE_DATE:=2019-08-15
+PKG_SOURCE_VERSION:=1cf8542aa64d816301ea286a8937d9b09ebcbc99
+PKG_MIRROR_HASH:=1cb20a749696852be0a512d51961365dd9c031362af0af1a2b9f5a3fb894885f
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/denisandroid/uPD72020x-Firmware.git
```

Comment:
Arg, the same problem! Look, it's not that I couldn't package the firmware for the WNDR4700, but the restrictive License the firmware has makes it impossible for OpenWrt to distribute it. :sob: 

For reference [License.rtf](https://github.com/denisandroid/uPD72020x-Firmware/blob/master/License.rtf)

 YOU AGREE TO BE BOUND BY THE TERMS AND CONDITIONS OF THIS EULA BY INSTALLING, COPYING OR OTHERWISE USING SOFTWARE.  IF YOU DO NOT AGREE THE TERMS AND CONDITIONS OF THIS EULA, DO NOT INSTALL, COPY, OR USE SOFTWARE.

 1. LICENSE
 1.1	You may install and/or use one copy of Software on your single computer (i) in which Software was pre-installed, (ii) with which the installer program of Software was bundled in a separate disc or (iii) specified by the manufacturer of  such computer (“Authorized Computer”).
 1.2	You may make one backup copy of Software, provided that your backup copy is not installed or used.

 2. RESTRICTIONS
 2.1	You may not install or use Software on any computer other than Authorized Computer.
 2.2	You may not modify, convert, change, combine or create derivative works of Software.
 2.3	You may not reverse engineer, reverse compile, disassemble or otherwise analyze Software.
 2.4	You may not delete or modify the copyright notice contained on or within Software.
 2.5	You may not sublicense, rent, distribute, assign, transfer or otherwise dispose of Software and its license to any third party.

Reply:
>Arg, the same problem! Look, it's not that I couldn't package the firmware for the WNDR4700, but the restrictive License the firmware has makes it impossible for OpenWrt to distribute it. 😭

Such a pity... 😢 

>"You may install and/or use one copy of Software on your single computer (i) in which Software was pre-installed,"

So, can I prepare script for easy download?

Reply:
I removed commit with firmware package.

Reply:
well, this resolves the issue.

As for easy download? Have you tried it?

In theory this "oneliner" should work:

    wget "http://github.com/denisandroid/uPD72020x-Firmware/raw/master/UPDATE.mem%202.0.2.6" -O /lib/firmware/renesas_usb_fw.mem

but it fails with (yes, this is on another device, I don't have a C200-V2):

   > wget: SSL support not available, please install one of the libustream-.*[ssl|tls] packages as well as the ca-bundle and ca-certificates packages.

so, because github insists on SSL/TLS, it needs the "right" libustream+ca-bundle. 
The next hurdle: xhci-pci! It uses firmware_request_nowarn, so any script has to either wait until the probe times out and reload the driver, use the sysfs firmware interface (in time) ... or soft reboot (the firmware file needs to be stored to nand).

this likely needs to be written down in the wiki or forum? what do you think?